redwood-tooling-codegen/src/main/kotlin/app/cash/redwood/tooling/codegen/widgetGeneration.kt

```diff
@@ -40,12 +40,29 @@ class ExampleWidgetSystem<W : Any>(
   override val Example: ExampleWidgetFactory<W>,
   override val RedwoodLayout: RedwoodLayoutWidgetFactory<W>,
 ) : WidgetSystem<W>,
-    ExampleWidgetFactoryProvider<W>,
-    RedwoodLayoutWidgetFactoryProvider<W>
+    ExampleWidgetFactoryOwner<W>,
+    RedwoodLayoutWidgetFactoryOwner<W> {
+  override fun visit(value: W, element: Modifier.UnscopedElement) {
+    Example.visit(value, element)
+    RedwoodLayout.visit(value, element)
+  }
```

Comment:
I think a future design for this is to give each schema a sealed unscoped modifier type. This would allow doing:
```kotlin
when (element) {
  is ExampleUnscopedModifier -> Example.visit(value, element)
  is RedwoodLayoutUnscopedModifier -> RedwoodLayout.visit(value, element)
  else -> throw!
}
```
to get throwing behavior on unknown modifiers.

It also means the `when` in each factory owner becomes exhaustive and gains an implicit throw (plus the inability to implement its `ExampleUnscopedModifier` interface).